Integrations/integration-Quinc.yml

```diff
@@ -0,0 +1,341 @@
+configuration:
+- display: ""
+  name: ServerName
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: ""
+  name: token
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: ""
+  name: scheme
+  defaultvalue: http
```

Comment:
no need for extra field called schema. User will have to fill the full url in the ServerName is enough. In display just set an example: `display: "Quin-C server url (e.g https://example.com:4443)"